internal/mysqld/parser/antlrparser/utils.go

```diff
@@ -0,0 +1,20 @@
+package antlrparser
+
+import (
+	"strings"
+
+	"github.com/antlr/antlr4/runtime/Go/antlr/v4"
+)
+
+func CaseInsensitiveEqual(s1, s2 string) bool {
+	return strings.ToLower(s1) == strings.ToLower(s2)
+}
+
+// GetOriginalText don't work.
```

Comment:
do you figure out why ?

Reply:
just see https://github.com/antlr/antlr4/issues/1302 for reference.